cypress/support/commands.js

```diff
@@ -0,0 +1,90 @@
+// These commands were sourced from the cypress-keycloak-login package
+const utils_1 = require('./utils');
+
+Cypress.Commands.add("kcGetToken", function (user) {
+  Cypress.log({ name: "Login" });
+  cy.fixture("users/" + user).then(function (userData) {
+    (userData.password)? null : userData.password = Cypress.env("kc_sa_pass")
+    let authBaseUrl = Cypress.env("auth_base_url");
+    let realm = Cypress.env("auth_realm");
+    let client_id = Cypress.env("auth_client_id");
+    cy.request({
+      method: "POST",
+      url: authBaseUrl + "/realms/" + realm + "/protocol/openid-connect/token",
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      followRedirect: false,
+      body: {
+        grant_type: 'password',
+        client_id: client_id,
+        client_secret: Cypress.env("kc_api_secret"),
+        username: userData.username,
+        password: userData.password,
+      }
+    })
+      .then(function (response) {
+        window.localStorage.setItem("TOKEN", response.body.access_token);
+        return response.body.access_token;
+      })
+  });
+});
+
+Cypress.Commands.add("kcLogout", function () {
+    Cypress.log({ name: "Logout" });
+    let authBaseUrl = Cypress.env("auth_base_url");
+    let realm = Cypress.env("auth_realm");
+    return cy.request({
+        url: authBaseUrl + "/realms/" + realm + "/protocol/openid-connect/logout"
+    });
+});
+
+Cypress.Commands.add("kcNavAs", function (user, visitUrl) {
+  if (visitUrl === void 0) { visitUrl = ""; }
```

Comment:
What's this checking for? If just wanting to make sure `visitUrl` is defined, we could probably use something simpler i.e. `visitUrl = visitUrl || ""`. This may raise linting errors if we apply linting, which I would recommend, as it is modifying a function param.